sprint_1/src/laura/uebungsprojekt/Main.java

```diff
@@ -12,6 +15,13 @@ public static void main(String[] args) {
         //UPLK80_Wetterstation.calc_temp_diff_of_two_days(temps);
         //UPLK80_Wetterstation.calc_temp_diff_of_two_days_oo(temps);
 
-        UPLK82_Schaltjahrprüfung.checkIfYearIsLeapYear();
+        //UPLK82_Schaltjahrprüfung.checkIfYearIsLeapYear();
+        Random dice = new Random();
+        int dice_one = dice.nextInt(7);
```

Comment:
Hier gibt es eine besser geeignete Datenstruktur, als die drei Würfel in drei verschiedenen Variablen zu speichern, die z.B. auch dann noch geeignet wäre, wenn du mehr als 3 Würfel haben möchtest. Hast du eine Idee, welche?